searchJob/views.py

```diff
@@ -1,7 +1,21 @@
 from django.shortcuts import render
-from.models import JobPost
+from .models import JobPost
 
 
 def jobs(request):
-    job_posts = JobPost.objects.all().order_by('creation_date')
+    if request.method == 'POST':
+        JobType = request.POST.get('JobType')
+        JobCity = request.POST.get('JobCity')
+        JobScope = request.POST.get('JobScope')
+        JobCompany = request.POST.get('JobCompany')
+    if JobCity == "":
+        JobCity = None
+    if JobCompany == "":
+        JobCompany = None
+    if JobType == "":
+        JobCity = None
+    if JobScope == "":
+        JobCompany = None
```

Comment:
what are these for?

Reply:
all the fields are optional, in case of empty field we should send None to the method.

Reply:
should they be optional? all these attributes seem important @bardielle

Reply:
the meaning is that during the search the user can decide which filters he wants, in the DB each JobPost contains all the attributes
![image](https://user-images.githubusercontent.com/69304551/103372134-e2e5c600-4ad9-11eb-84e4-61da4ace3633.png)
![image](https://user-images.githubusercontent.com/69304551/103372226-1aed0900-4ada-11eb-8653-04d8dee3609c.png)

Reply:
you're right, I didn't notice it was part of the search (I thought it was part of job post).

But I think there's a better way of doing it rather than converting empty strings to None.

Reply:
I tried to insert None as default value in the input tag (HTML) , and to search for other ways..but couldn't find a better solution yet.

Reply:
But if, say, `JobCity = ""` (empty string) , in the [search function](https://github.com/beyond-io/joboard/blob/672960c7bd191dffcfda45d5a60930d8817ec80c/searchJob/models.py#L62)
you have `if JobCity:`, and it should be false and not enter the IF. 

so it should work even if you don't convert it to None - can you please recheck?

Reply:
You're right! thx :)

Done